tests/ui/autodiff/autodiff_illegal.rs

```diff
@@ -30,16 +30,24 @@ pub fn f4(x: f64) {
     unimplemented!()
 }
 
+
+#[autodiff(df5, Forward, Dual, Dual)]
+fn f5(x: f32, y: f32) -> f32 {
+    //~^^ ERROR     expected 3 activities, but found 2
+    x + y
+}
+
```

Comment:
Please avoid such churn, if you want to insert it here in between and keep the naming scheme use something like `f4_1`